api/pom.xml

```diff
@@ -181,6 +181,15 @@
                     <links>
                         <link>http://docs.oracle.com/javase/6/docs/api/</link>
                     </links>
+                    <description>Jakarta WebSocket API documentation</description>
+                    <doctitle>Jakarta WebSocket API documentation</doctitle>
+                    <windowtitle>Jakarta WebSocket API documentation</windowtitle>
+                    <header><![CDATA[<br>Jakarta WebSocket API v${project.version}]]></header>
+                    <bottom><![CDATA[
+  Comments to: <a href="mailto:websocket-dev@eclipse.org">websocket-dev@eclipse.org</a>.<br>
+  Copyright &#169; 2017-{currentYear} - <a href="https://www.eclipse.org">Eclipse Foundation</a>. All rights reserved.<br>
```

Comment:
I think it should be 2012.

Reply:
I agree with 2012 as the start date.

Reply:
No thoughts on the fact that there's no Oracle copyright present here?

Reply:
Change the botton to this:
```
<bottom><![CDATA[
Copyright &#169; {inceptionYear}&#x2013;{currentYear} Oracle and/or its affiliates. All rights reserved.<br>
Use is subject to <a href="{@docRoot}/doc-files/EFSL.html" target="_top">license terms</a>.]]>
</bottom>
```
Override `inceptionYear' from the parent pom and set it's value to 2012.

It should fix everything.

Reply:
Fixed in commit 5599127ca22c67dd23078030f522b51524d4cfbe